omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -61,7 +61,23 @@ public String getAttribute(Person person, String attributeTypeUuid) {
 		
 		return null;
 	}
-	
+	/**
+	 * @since 1.15.0
+	 * Gets the person Attribute display value   for the specified person attribute type uuid
```

Comment:
Is there a reason why you are using capital A for Attribute?
And for the double space after `display value`?

Reply:
It was an oversight on my side, it has been corrected